model/github.py

```diff
@@ -201,3 +205,26 @@ def _optional_attributes(self):
             'secondary_authToken',
             'recovery-codes',
         )
+
+
+def find_github_cfg_for_api(
+    api_url: str | collections.abc.Iterable[str],
```

Comment:
hmm... I am not fully happy with this helper function. Firstly, it has some overlaps w/ ccc.github's lookup by repo-url, secondly it mixes two aspects (lookup for _one_ vs lookup first matching cfg).

Do you think we might merge the required functionality into ccc.github (or mv from ccc.github into model/github)?